examples/tensorflow/requirements.txt

```diff
@@ -2,6 +2,6 @@ addict
 absl-py==0.10
 tensorflow_datasets==4.2.0
 tensorflow_hub
-tensorflow_addons~=0.16.1
+tensorflow_addons~=0.19.0
```

Comment:
According to https://github.com/tensorflow/addons#python-op-compatibility-matrix